src/pages/Invoice/AddItemsModal.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-unused-vars */
+import React, { Fragment } from "react";
+import PropTypes from "prop-types";
+import Modal from "react-modal";
+import { Link } from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+Modal.setAppElement("*");
+
+const AddItemsModal = ({
+    modalStatus,
+    setModalStatus,
+    itemInfo,
+    invoiceRecipentDetails,
+    setInvoiceRecipentDetails,
+}) => {
+    const addItem = (id, name, price) => {
```

Comment:
useCallback

Reply:
Fixed this one

Reply:
@smilegupta i don't see your changes here

Reply:
i forgot to push the changes to remote... now it should be visible

Reply:
you should use `React.useCallback` hook